components/combobox/src/auro-combobox.js

```diff
@@ -569,8 +653,13 @@ export class AuroCombobox extends LitElement {
         }
       }
 
-      if (evt.key === 'Tab') {
+      if (evt.key === 'Tab' && this.dropdown.isPopoverVisible) {
```

Comment:
```suggestion
      if (evt.key === 'Tab') {
```

The hideBib() function already does a check for `this.dropdown.isPopoverVisible`